src/spikeinterface/extractors/neoextractors/neuralynx.py

```diff
@@ -22,22 +22,24 @@ class NeuralynxRecordingExtractor(NeoBaseRecordingExtractor):
         If there are several streams, specify the stream name you want to load.
     all_annotations: bool, default: False
         Load exhaustively all annotations from neo.
+    exlude_filename: list[str], default: None
+        List of filename to exclude from the loading.
```

Comment:
```suggestion
        List of filename to exclude from the loading.
        For example, use `exclude_filename=[".nev"]` to skip loading the event files.
```
Not sure this is accurate, but something along these lines ;)

Reply:
I am not sure this is working I think you need to provide a full filename but a will test it asap thank you

Reply:
Yes, Alessio nailed it. Something like that. It can be a docstring example if you prefer that.

I think this will make it easier for users to identify how and why to use this parameter : )